### PR TITLE
[PREVIEW] RDM-2542: GDPR - Probate solicitors footer links

### PR DESCRIPTION
**Pull request by MKaramath**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-2542

### Change description ###
To add the following navigation in footer links

Cookies
Privacy Policy
Terms and Conditions
Contact Us

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/app/footer-nav/contact-us.component.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+})
+export class ContactUsComponent implements OnInit {
+
+  constructor(public router: Router,
+              private route: ActivatedRoute,
+              private location: Location) { }
+
+  ngOnInit() {
+  }
+
```
Comment (adr1ancho) [inline src/app/footer-nav/contact-us.component.ts]: Question Do we need OnInit and ngOnInit?

File: src/app/footer-nav/cookies.component.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+})
+export class CookiesComponent implements OnInit {
+
+  constructor(public router: Router,
+              private route: ActivatedRoute,
+              private location: Location) { }
+
+  ngOnInit() {
+  }
+
```
Comment (adr1ancho) [inline src/app/footer-nav/cookies.component.ts]: Question do we need OnInit and ngOnInit

File: src/app/shared/accordion/accordion.component.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+})
+export class AccordionComponent implements OnInit {
+
+  @Input() title: string;
+  isHidden = true;
+
+  @Input() lastAccordion: boolean;
+
+  constructor() { }
+
```
Comment (adr1ancho) [inline src/app/shared/accordion/accordion.component.ts]: Do we need constructor here

File: src/app/footer-nav/contact-us.component.ts (hunk 1 above)
Comment (MKaramath) [reply src/app/footer-nav/contact-us.component.ts]: Removed them

File: src/app/footer-nav/cookies.component.ts (hunk 2 above)
Comment (MKaramath) [reply src/app/footer-nav/cookies.component.ts]: Removed them.

**Comment by Saravanabh** [comment]
Testing completed and ready to be merged.

**Pull request merged**